openlibrary/plugins/upstream/utils.py

```diff
@@ -854,6 +855,50 @@ def today():
     return datetime.datetime.today()
 
 
+class HTMLTagRemover(HTMLParser):
+
+    def __init__(self):
+        super().__init__()
+        self.data = []
+
+    def handle_data(self, data):
+        self.data.append(data)
+
+
+@public
+def reformat_html(html_str, max_length=None):
+    """
+    Reformats an HTML string, removing all opening and closing tags.
+    Adds a line break element between each set of text content.
+    Optionally truncates contents that exceeds the given max length.
+
+    returns: A reformatted HTML string
```

Comment:
Let's add some tests (either docstring or in a file) to this one; this is the kind of unit-function that will benefit a lot from having a few simple tests :)

```suggestion
```

Reply:
Added unit tests.